modules/cert-manager-override-scheduling.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-override-scheduling_{context}"]
+= Configuring scheduling overrides for cert-manager Operator components
+
+After installing the {cert-manager-operator}, you can configure the pod scheduling from the {cert-manager-operator} API for the {cert-manager-operator} components such as {cert-manager-operator} controller, CA injector, and Webhook. Configuring the `overrideScheduling` block in the `certmanager.operator` custom resource enables you to define `nodeSelector` and `tolerations` for the `controllerConfig`, `webhookConfig`, and `cainjectorConfig` sections respectively.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have installed the {cert-manager-operator} 1.15.0 or later.
+
+.Procedure
+
+* To configure scheduling overrides, modify the `certmanager.operator` custom resource. Use the `overrideScheduling` field within `controllerConfig`, `webhookConfig`, or `cainjectorConfig` to specify `nodeSelector` and `tolerations` for overriding scheduling of {cert-manager-operator} `controller`, `webhook` or `cainjector` pods respectively.
++
+.Example output
```

Comment:
It's a part of the `Procedure`

```suggestion
```

Reply:
+1 this needs to be removed since this is not an output, as far as I can tell, this is the actual command that users need to run